bio_hansel/metadata.py

```diff
@@ -36,7 +36,7 @@ def read_metadata_table(path: str) -> Optional[pd.DataFrame]:
             list(FILE_EXT_TO_PD_READ_FUNC.keys())
         ))
         return None
-    dfmd = FILE_EXT_TO_PD_READ_FUNC[file_ext](path)  # type: pd.DataFrame
+    dfmd = FILE_EXT_TO_PD_READ_FUNC[file_ext](path, sep='\t')  # type: pd.DataFrame
```

Comment:
This will break on CSV files if you set the delimiter to tab. A better solution might be creating [partial functions](https://docs.python.org/2/library/functools.html#functools.partial) in `FILE_EXT_TO_PD_READ_FUNC` for tab-delimited files.

Reply:
I'll look into it.

Although, I've just retested it using a .csv file and a .tsv file and nothing seems off, the data is being included no problem

Reply:
[Please see the following Jupyter Notebook](https://gist.github.com/peterk87/23fca3fa949777fa5cd5b9430d03985d)